pkgs/top-level/all-packages.nix

```
@@ -1513,6 +1513,8 @@ with pkgs;
 
   azure-cli-extensions = recurseIntoAttrs azure-cli.extensions;
 
+  azure-sdk-for-cpp = callPackage ../development/libraries/azure-sdk-for-cpp { };
```

We need Hydra to build it and make it available in the binary cache.

```suggestion
  azure-sdk-for-cpp = recurseIntoAttrs (callPackage ../development/libraries/azure-sdk-for-cpp { });
```